### ISSUE TITLE
Flatpak version too old runtimes

### ISSUE THREAD

**Issue by tgodtaylor** [hbons/SparkleShare#1961]
Too old runtimes which need to migrate.

like org.gnome.platform and freedesktop

**Comment by tymophy** [comment]
Was just coming to open this exact issue...

**Comment by hbons** [comment]
You're right. When I bumped the runtimes I got errors compiling mono in the build service. I need to investigate... help welcome!
